python/tests/README.md

```diff
@@ -10,6 +10,15 @@ can be run against a live Nessie engine.
    `./mvnw quarkus:dev -am -pl :nessie-quarkus` locally.
 1. Run `pytest --record-mode=all tests/` (in a venv with `requirements_dev.txt` installed)
    to add test data for newly added tests and updated tests data.
+   * Note: when running `test_nessie_auth.py` in recording more, the tests will most likely fail out-of-the box
```

Comment:
same for `test_nessie_cli_auth.py`

Reply:
updated: a18b0aff916541eff14766b60b9dfdede0270d2f